### PR TITLE
Community section: add link to GH Discussions

### PR DESCRIPTION
**Pull request by mewzherder**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ Components:
 The GitOps Toolkit is always looking for new contributors and there are a multitude of ways to get involved. Depending on what you want to do, some of the following bits might be your first steps:
 
 - Join our upcoming dev meetings ([meeting access and agenda](https://docs.google.com/document/d/1l_M0om0qUEN_NNiGgpqJ2tvsF2iioHkaARDeh6b70B0/view))
+- Ask questions and add suggestions in our [GitOps Toolkit Discussions](https://github.com/fluxcd/toolkit/discussions)
```
Comment (dholbach) [inline docs/index.md:63]: It's already mentioned two lines below..?

Comment (dholbach) [reply docs/index.md:63]: Maybe it should be clearer?

Comment (mewzherder) [reply docs/index.md:63]: k. Can we remove this part and change the text below to - Ask questions and add suggestions in our [GitOps Toolkit Discussions](https://github.com/fluxcd/toolkit/discussions)

Below only says "planning discussions" but my understanding is that the GH Discussions is so much more than that and my text was intended to drive more people to the GH Discussions page where can ask questions, get help, _and_ make suggestions / be part of planning. Does that make the intention of this change more clear?

Comment (scottrigby) [reply docs/index.md:63]: That is definitely true per the new Flux community governance https://github.com/scottrigby/flux-community/blob/governance/GOVERNANCE.md#proposal-process.

**Comment by hiddeco** [comment]
@mewzherder we require a DCO sign-off since last Friday, please follow [the instructions here](https://github.com/fluxcd/flux2/pull/365/checks) to make the tests pass :sunflower:

**Review by scottrigby — APPROVED** [review]
Nice

**Comment by stealthybox** [comment]
This is good to merge :+1:

**Review by stefanprodan — APPROVED** [review]

**Pull request merged**